src/ch/ehi/interlis/domainsandconstants/basetypes/CoordinateType.java

```diff
@@ -295,5 +295,22 @@ public void setGeneric(boolean generic) {
                 
                 // -end- 394886B10363 detail_end "CoordinateType"
 
+                private boolean isMulti;
+
+                public boolean isMulti() {
+                  return isMulti;
+                }
+
+                public void setMulti(boolean isMulti) {
```

Comment:
richtig wäre:

    if(isMulti != value1){
      isMulti= value1;
      ch.ehi.uml1_4.changepropagation.MetaModel.getInstance().notifyChange(new ch.ehi.uml1_4.changepropagation.MetaModelChange(this,"setMulti"));
    }